exercises/CMakeLists.txt~

```diff
@@ -0,0 +1,47 @@
+###############################################################################
```

Comment:
This file shouldn't have been committed.

Reply:
Thanks, fixed.